### PR TITLE
feat(EMS-3565): iterations - Eligibility - Eligible to apply online - Content

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR adds content to the Eligible to apply online page

## Resolution :heavy_check_mark:
* Added content to page
* Added new link for privacy notice
* Updated e2e test

### CODE REVIEW

File: e2e-tests/content-strings/links.js [hunk 1]
```diff
@@ -31,5 +31,6 @@ export const LINKS = {
     BRIBERY_ACT_2010_GUIDANCE: 'https://www.justice.gov.uk/downloads/legislation/bribery-act-2010-guidance.pdf',
     ICO_MAKE_A_COMPLAINT: 'https://ico.org.uk/make-a-complaint',
     COMPANIES_HOUSE: 'https://find-and-update.company-information.service.gov.uk',
+    PRIVACY_NOTICE: 'https://www.gov.uk/government/publications/ukef-privacy-notice/ukef-privacy-notice--2',
```
Comment (ttbarnes) [inline e2e-tests/content-strings/links.js:34]: We already have a content string for this, `PRIVACY`, in the same file

File: src/ui/server/content-strings/pages/insurance/eligibility/index.ts [hunk 2]
```diff
@@ -130,6 +130,17 @@ const ELIGIBLE_TO_APPLY_ONLINE = {
...
+    INTRO: "We're committed to protecting the privacy and security of the personal information we hold. Read our",
+    PRIVACY_NOTICE: {
+      LINK: {
+        TEXT: 'privacy notice',
+        HREF: LINKS.EXTERNAL.PRIVACY_NOTICE,
+      },
+      TEXT: 'for more details.',
+    },
+  },
+  CONTINUE_SUBMIT: 'Once you continue your application you will have 30 days to submit it.',
```
Comment (ttbarnes) [inline src/ui/server/content-strings/pages/insurance/eligibility/index.ts:143]: Can we use a constant for 30, maybe `SUBMISSION_DEADLINE_IN_DAYS`, like `SUBMISSION_DEADLINE_IN_MONTHS`.

Would be nice to also find and replace the other few instances of "30 days" with the new constant

File: e2e-tests/pages/insurance/eligibility/eligibleToApplyOnline.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 const eligibleToApplyOnlinePage = {
   insetText: () => cy.get('[data-cy="inset-text"]'),
+  privacyIntro: () => cy.get('[data-cy="privacy-intro"]'),
+  privacyNotice: () => cy.get('[data-cy="privacy-notice"]'),
+  privacyNoticeLink: () => cy.get('[data-cy="privacy-notice-link"]'),
```
Comment (ttbarnes) [inline e2e-tests/pages/insurance/eligibility/eligibleToApplyOnline.js:5]: maybe object structure?


```ts
privacy: {
  intro: () =>
  notice: () => 
},
```

File: src/ui/server/content-strings/pages/insurance/eligibility/index.ts (hunk 2 above)
Comment (Zainzzkk) [reply src/ui/server/content-strings/pages/insurance/eligibility/index.ts]: done

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**